Gruntfile.js

```diff
@@ -474,6 +474,25 @@ module.exports = function(grunt) {
                 reloadOnRestart: true,
                 watchTask: true
             }
+        },
+
+        validation: {
+            options: {
+                reset: grunt.option('reset') || false,
+                stoponerror: false,
+                maxTry: 3,
+                relaxerror: ['Bad value X-UA-Compatible for attribute http-equiv on element meta.'], // ignores these errors 
+                generateReport: true,
+                errorHTMLRootDir: "tests/validation",
+                useTimeStamp: true,
+                errorTemplate: "tests/validation/w3c_validation_error_Template.html"
+            },
+            files: {
+                src: ['dist/views/*.html', 
```

Comment:
I think we should list the relevant lexicon urls we'd want to validate rather than just the `.html` files e have in `/dist`, this will let us validate against our Twig output.